python/aws_lambda_powertools/tracing/tracer.py

```diff
@@ -6,106 +6,136 @@
 
 from aws_xray_sdk.core import models, patch_all, xray_recorder
 
+from ..helper.register import RegisterMeta
+
 is_cold_start = True
 logger = logging.getLogger(__name__)
 logger.setLevel(os.getenv("LOG_LEVEL", "INFO"))
 
 
-class Tracer:
+class Tracer(metaclass=RegisterMeta):
     """Tracer using AWS-XRay to provide decorators with known defaults for Lambda functions
 
-    When running locally, it honours POWERTOOLS_TRACE_DISABLED environment variable
+    When running locally, it honours `POWERTOOLS_TRACE_DISABLED` environment variable
     so end user code doesn't have to be modified to run it locally
     instead Tracer returns dummy segments/subsegments.
 
     Tracing is automatically disabled when running locally via via SAM CLI.
 
-    It patches all available libraries supported by X-Ray SDK
+    By default, it patches all available libraries supported by X-Ray SDK. \n
```

Comment:
__Typo__: `\n` at the end of the line

Reply:
Not a typo - `pdoc3` uses this to enforce a newline as the link below got crammed